spec/puppet-lint/configuration_spec.rb

```diff
@@ -49,7 +49,10 @@
   end
 
   it 'should be able to set sane defaults' do
-    subject.defaults
+    override_env do
+      ENV.delete('GITHUB_ACTION')
+      subject.defaults
+    end
```

Comment:
Annotating my own code: technically this is redundant now that there's `ENV.delete()` in `spec_helper.rb`, but it doesn't hurt to be explicit.